docs/build_windows.md

```diff
@@ -14,7 +14,7 @@ Then, run a static build using the mingw compiler:
     export CROSS_COMPILE=x86_64-w64-mingw32
     export CC=$CROSS_COMPILE-gcc
     ./autogen.sh    # only necessary if building from a git source tree
-    ./build_pkg.sh
+    ./scripts/build_pkg.sh
 
-The script should exit successfully and you should have an `fwup.exe` binary. To
-recompile from now on, it is sufficient to just run `make`.
+The script should exit successfully and you should have an `fwup.exe` binary.
+To recompile, assuming these environment variables are still in place, it is sufficient to just run `make`.
```

Comment:
Ah, good point, You're right. I had been re-running `build_pkg.sh` during development, so I still had that stuck in my mind.